### ISSUE TITLE
Edge case with update/create-buildpack when buildpack has nil stack

### ISSUE THREAD

**Issue by idoru** [cloudfoundry/cli#1380]
## Command  

Given there is a buildpack named foo_buildpack with stack cflinuxfs2 AND a buildpack named foo_buildpack with a NIL stack, even with the changes from https://github.com/cloudfoundry/cli/pull/1320 it's not possible to delete or update the buildpack with the NIL stack, since there is no way to pass `nil` to the `-s` flag

```
cf update-buildpack foo_buildpack -s ????
cf delete-buildpack foo_buildpack -s ????
```

## What occurred

Can't specify criteria to delete/update buildpack with NIL stack

## What you expected to occur

Have some way to specify the edge case where buildpack has a nil stack. Maybe `--nil-stack` ?

## CLI Version  

master + https://github.com/cloudfoundry/cli/pull/1320

## CC API Endpoint Version  

2.110.0 / 3.45.0
CAPI Release: 1.56.0

## Platform & Shell Details  

ALL

## Any other relevant information  

Note that to implement this selection logic, the CLI will need to retrieve the list of all buildpacks from CAPI since it does not support filtering results where a field is nil.

**Comment by idoru** [comment]
FWIW we are adding a flag to the libbuildpack buildpack-packager that is used to set the stack internally in a buildpack's manifest, this is what ultimately controls the value of the stack when the buildpack is uploaded to CC. Since CC's behavior when the stack is `nil` is to provide the buildpack to all stacks we've decided to initially name our flag `-any-stack` (see: https://github.com/cloudfoundry/libbuildpack/commit/9dc5fa4beda1526e0d18b4dffb47c11b087e7d44)

It might make sense to use consistent naming with whatever flag we introduce into the CLI to handle this. Happy to discuss and change ours if we want to go with a better name that works universally across our domains.

**Comment by abbyachau** [comment]
Hi @idoru since the CLI team is working through the buildpacks refactors and the stack association stories in this epic: https://www.pivotaltracker.com/story/show/158348569, could we close this issue?

**Comment by idoru** [comment]
Absolutely! We had originally filed just this to make sure it didn’t slip
through the cracks.
